src/Castle.Windsor.Extensions.DependencyInjection/WindsorScopeFactory.cs

```diff
@@ -16,29 +16,34 @@
 namespace Castle.Windsor.Extensions.DependencyInjection
 {
 	using System;
-	
+
 	using Castle.Windsor;
 	using Castle.Windsor.Extensions.DependencyInjection.Scope;
-	
+
 	using Microsoft.Extensions.DependencyInjection;
 
 	internal class WindsorScopeFactory : IServiceScopeFactory
 	{
 		private readonly IWindsorContainer scopeFactoryContainer;
+		private readonly ExtensionContainerRootScope rootScope;
```

Comment:
Field `rootScope` is unused.

Reply:
![image](https://user-images.githubusercontent.com/2411563/101459632-c7532980-3938-11eb-8326-f614f26887f1.png)

Reply:
cleaned up now